update OSB's redline testing documentation (#10185)
opensearch-project/documentation-website: _benchmark/reference/commands/redline-test.md

## Latency or CPU-based Feedback
OSB supports a `timeout` value per request, which tells the system to cancel a request if it is taking longer than the specified time. The flag `--client-options=timeout:<int>` allows users to specify this time period. The default value is 10 seconds.

You can increase or decrease this value to customize the maximum latency tolerated by OSB during a redline test as well. 
For example, if you want to find the maximum load your cluster can handle without bringing latency above 15 seconds, setting the timeout value in client options to 15 will achieve this.

Redline testing now supports CPU-based feedback in addition to request error and latency monitoring. This helps avoid pushing your cluster past its safe utilization limits. 

### Requirements
- A metrics store must be configured for CPU-based feedback. Using an in-memory store will result in an error:
```bash
[ERROR] Cannot execute-test. Error in worker_coordinator (CPU-based feedback requires a metrics store. You are using an in-memory metrics store)
```
The `--redline-cpu-max-usage` flag is required. This sets the maximum CPU usage percentage (per node) allowed during the test.

> The `node-stats` telemetry device is automatically enabled when using this feature.

Reviewer: Nit: Add bullets here (`-`)

```
{% include copy.html %}

## Latency- or CPU-based feedback

> OpenSearch Benchmark (OSB) supports a `timeout` value per request, which cancels a request if it exceeds the specified duration. You can set this value using the `--client-options=timeout:<int>` flag. The default is 10 seconds.

Reviewer 2: ```suggestion
OpenSearch Benchmark supports a `timeout` value per request, which cancels a request if it exceeds the specified duration. You can set this value using the `--client-options=timeout:<int>` flag. The default is 10 seconds.
```

> You can adjust this value to define the maximum request latency OSB should tolerate during redline testing. For example, to determine the highest load your cluster can handle without exceeding 15 seconds of latency, set the timeout in client options to `15`.

Reviewer 2: ```suggestion
You can adjust this value to define the maximum request latency that OpenSearch Benchmark should tolerate during redline testing. For example, to determine the highest load your cluster can handle without exceeding 15 seconds of latency, set the timeout in client options to `15`.
```

> Redline testing also supports CPU-based feedback in addition to latency and request error monitoring. This helps prevent exceeding safe utilization limits for your cluster.

Reviewer 2: ```suggestion
Redline testing also supports CPU-based feedback in addition to latency and request error monitoring. This helps to prevent exceeding safe utilization limits for your cluster.
```

- `--redline-scale-step`: Specifies the number of clients to unpause in each scaling iteration.
- `--redline-scaledown-percentage`: Specifies the percentage of clients to pause when an error occurs.
- `--redline-post-scaledown-sleep`: Specifies the number of seconds the feedback actor waits before initiating a scale-up after scaling down.
- `--redline-max-clients`: Specifies the maximum number of clients allowed during redline testing. If unset, OpenSearch Benchmark defaults to the number of clients defined in the test procedure.

### For CPU-based Feedback
- `--redline-cpu-max-usage`: (Required) Max allowed CPU load (%) per node before triggering a scale-down.
- `--redline-cpu-window-seconds`: Duration (in seconds) over which to average CPU usage per node (default: 30).

> - `--redline-cpu-check-interval`: Interval (in seconds) between CPU usage checks (default: 30).

Reviewer: Should we just mark these as optional and remove the `Optional Tuning Flags` section above as they're redundant?
